specification/security/resource-manager/Microsoft.Security/preview/2021-01-01-preview/secureScore.json

```diff
@@ -0,0 +1,640 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2021-01-01-preview"
```

Comment:
for the future, when adding a new api-version your first commit should be the content from the previous api-version unchanged. Subsequent commits should be for the new api-version. This way diffing the first and last commit make it easy for the reviewer to determine what changed. It took me a while to figure out which api-version you were changing from - I had to look at the existing swaggers and your manifest. I had to then diff the files locally after checking out your branch - this is a slow week so I am able to spend more time on this review - however usually the reviewer will not have the time to do all this. So, please follow my suggestion above for the future.

Reply:
Will do.
Thanks